src/routes/welcome/gettingStarted.tsx

```diff
@@ -59,21 +79,22 @@ export default function GettingStarted({ page }) {
           <PageWrapper style={{ height: contentSize }}>
             <AnimatePresence initial={false}>
               <Page key={page} ref={contentRef}>
-                {gettingStartedPages[page - 1]}
+                <View />
               </Page>
             </AnimatePresence>
           </PageWrapper>
         </Content>
         <Footer>
           <PageIndicatorContainer>
-            {gettingStartedPages.map((_, i) => (
+            {Views.map((_, i) => (
               <PageIndicator
-                onClick={() => navigate(i + 1)}
+                key={i}
                 active={page === i + 1}
+                onClick={() => navigateToPage(i + 1)}
```

Comment:
This could be replaced with `navigate(i + 1)`. See https://github.com/arconnectio/ArConnect/issues/568